lib/snowflake/index.js

```diff
@@ -0,0 +1,60 @@
+const EventEmitter = require('events').EventEmitter
+const snowflake = require('snowflake-sdk')
+const Log = require('@kelchy/log')
+
+class Snowflake extends EventEmitter {
+  constructor (snowflakeOpts = {}, options = {}) {
+    // if application developer initializes its own logger
+    // and pass it to the constructor i.e. they need logging to stdout
+    // it should have 3 functions out, debug and error, if not, do nothing
+    const log = Log.isValid(options.log) ? options.log : new Log.Empty()
```

Comment:
can we use Log.ErrorOnly as default? so potential errors from this module will definitely go to stderr